ynput/OpenPype
# Fusion: Project/User option for output format (create_saver)

## Changelog Description
Adds "Output Image Format" option which can be set via project settings and overwritten by users in "Create" menu. This replaces the current behaviour of being hardcoded to "exr". Replacing the need for people to manually edit the saver path if they require a different extension.

## Additional info
I've removed some potential bloat `saver["OutputFormat"] = file_format`. In my testing Fusion is able to infer this information based on the file extension (Fusion17 and 18).

## Testing notes:
1. Launch OP Project settings
2. Tweak the new option "Output Image Format" (see screenshot below)
<img width="533" alt="image" src="https://github.com/ynput/OpenPype/assets/20257044/e5766ffd-9c02-4f49-81bf-72d36645af57">

3. Launch project in Fusion
4. Open "Create" menu
5. Check that "Output Image Format" matches your new default 
6. Create and validate that it has respected your format (also check that it infers the correct "Output Format" etc)
